packages/quantic/force-app/main/default/lwc/quanticResultQuickview/quanticResultQuickview.js

```diff
@@ -121,10 +121,6 @@ export default class QuanticResultQuickview extends LightningElement {
       );
       this.dispatchEvent(resultActionRegister);
     }
-    this.addEventListener(
```

Comment:
Is there any reason why we were adding it via code instead of html event listener?

Wasn't there something weird with double `__` in HTML event listeners? I vaguely remember an issue related to this.?

Reply:
I think it was just an error that we added it via a listener while it was already added in the HTML.
and no I don't find any issue with the `__` in the HTML, it's used in a lot of our events.